crates/bdk/examples/compiler.rs

```diff
@@ -42,12 +42,23 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     // Create a `wsh` type descriptor from the policy.
     // `policy.compile()` returns the resulting miniscript from the policy.
-    let descriptor = Descriptor::new_wsh(policy.compile()?)?;
+    let descriptor = Descriptor::new_wsh(policy.compile()?)?.to_string();
 
-    println!("Compiled into following Descriptor: \n{}", descriptor);
+    println!("Compiled into Descriptor: \n{}", descriptor);
```

Comment:
We are using edition 2021, so you can do:
```suggestion
    println!("Compiled into Descriptor: \n{descriptor}");
```

Reply:
I opened #1394 to discuss code style.